src/puppetlabs/puppetdb/command/dlo.clj

```diff
@@ -71,31 +76,43 @@
                           (.printStackTrace exception out))
                         attempts))))
 
+(defn- strip-metadata-suffix
+  [queue-metadata-str]
+  (str/replace queue-metadata-str #"\.json$" ""))
+
 (defn- entry-cmd-data-filename
   [entry]
-  ;; 10291-1469469689-cat-4-foo.org.json
+  ;; 10291_1469469689_cat_4_foo.org.json
```

Comment:
Ok. Somewhat regrettable for us since it means DLO filenames will have two different kinds of separators, but probably too much work to change it at this point.